chia/_tests/core/ssl/test_ssl.py

```diff
@@ -37,6 +42,41 @@ async def establish_connection(server: ChiaServer, self_hostname: str, ssl_conte
         await wsc.close()
 
 
+#
+# This is needed on linux and mac when running in asyncio debug otherwise
+# The SSL Error causes the test to exit and fail prematurely in an uncatchable way
+# This doesn't seem to work on Windows as the exception handler is never called
+# at least for this SSL failure
+#
+@contextlib.contextmanager
+def ignore_ssl_cert_error():
+    current_loop = asyncio.get_event_loop()
+    old_handler = current_loop.get_exception_handler()
+    old_handler_fn = old_handler  # or lambda _loop, ctx: loop.default_exception_handler(ctx)
```

Comment:
It doesn't seem like we need both of these names.

Reply:
changed